### DISCUSSION TITLE
限時活動的倒數時間怎麼在主頁面顯示

### DISCUSSION THREAD

**Discussion by WhiteCloud0106** [Lost-MSth/Arcaea-server#180] [Q&A] [answered]
如題，就是像官方限時活動正在進行時主頁面的“world”那格旁邊會有一個藍色框框限時幾天或幾小時之類的
這幾個月進行了幾個限時活動，但好像主頁面都不會顯示，只有相關歌曲的旁邊會顯示而已

**Comment by Lost-MSth** [comment, marked as answer]
设置里面有个 `AVAILABLE_MAP: list[str]`，可以填写 event 地图的 map_id

这设计确实挺不好的，但我懒得想怎么改了（）
